src/sentry/deletions/defaults/group.py

```diff
@@ -24,6 +24,11 @@
 
 logger = logging.getLogger(__name__)
 
+GROUP_CHUNK_SIZE = 100
```

Comment:
@markstory It's 100 in the delete_group's task but 1000 within this custom task (see screenshot).

Let's keep it at 100 (scheduled deletions for projects will now be doing 100 groups at a time instead of 1000 groups).

<img width="645" height="116" alt="Image" src="https://github.com/user-attachments/assets/7c9a5dde-8fe0-4115-aaa3-26183eda2c50" />